web-app/src/components/tournament/details/MatchComponent.vue

```diff
@@ -29,12 +29,12 @@
         style="width: 100px"
         class="q-ma-auto q-mx-sm column self-center justify-center"
       >
-        <!-- <div class="matchDate">{{ matchFormatedDate }}</div> -->
+        <div class="matchDate">{{ matchFormatedDate }}</div>
         <div style="text-align: center">
           <div v-if="!getMatchStatus" class="score q-mx-auto">
```

Comment:
Bad class name. Outside of any CSS standard

Reply:
But here we did not agree on any CSS class naming convention for this project. Otherwise, you could come up with an idea

Reply:
I came up from the begining. BEM methodology.

Reply:
Yes, you are using BEM, but we didn't talk to use only that

Reply:
but we haven't all agreed to it, you're the only one using it in the project, all the views prepared by @Zummek  and I don't use it.  We can change it in all front if you want so bad but again, this is not a task to go through all front files